### PR TITLE
Version script

### PR DESCRIPTION
**Pull request by stephanebisson**
* Keep all versions up to date when bumping the package version with `npm version <part>`
* Update publish script to also publish the new version under tags/

### CODE REVIEW

**Comment by hueitan** [comment]
I also notice the version in `package-lock.json`  doesn't update.

https://github.com/wikimedia/wikipediapreview-wordpress/blob/0aabc0fecba99d4ee43127e59799e842bcd9e645/package-lock.json#L2-L3

**Comment by stephanebisson** [comment]
It will stay up to date when we handle version correctly with npm version. I have fixed it on main for now.

File: wikipediapreview.php [hunk 1]
```diff
@@ -11,23 +11,25 @@
  * License: MIT
  * License URI: https://github.com/wikimedia/wikipedia-preview/blob/main/LICENSE
  */
+
+DEFINE( 'WIKIPEDIA_PREVIEW_PLUGIN_VERSION', '1.0.2' );
```
[the remark this answers is not in the document]
Comment (hueitan) [reply wikipediapreview.php:15]: no should be the version in `package.json`

**Comment by hueitan** [comment]
`npm run version` works for me, i didn't test `npm run postvresion` as it push a new tag.

**Pull request merged**